### DISCUSSION TITLE
how to prevent some url from triggering the loading bar?

### DISCUSSION THREAD

**Discussion by d-mok** [quasarframework/quasar#10498] [General - Components / Directives / etc]
I used the loading bar plugin and it works fine. The problem is that the loading bar is triggered on every ajax call. However, I'm using a frontend monitoring service (logrocket) which will automatically send data to their server every few seconds. As a result, the loading bar is triggered every few seconds. How can I tell it to skip the ajax calls to this particular url? Or how can I tell it to distinguish 'normal ajax calls' and 'ajax call to logrocket'?

**Comment by MilosPaunovic** [comment]
You could use `skip-hijack` prop on loading bar as explained (here) [https://quasar.dev/vue-components/ajax-bar#qajaxbar-api] but then you have to manually trigger loading bar when you need it.
